code/game/gamemodes/devil/devilinfo.dm

```diff
@@ -435,5 +435,6 @@ var/global/list/lawlorify = list (
 	soulCounter = SOULVALUE
 	var/obj/screen/devil/soul_counter/K = new()
 	K.update()
+	var/mob/living/carbon/D = new()
```

Comment:
Dude, it works, that;s fucking amazing. One last question, I also made a update() proc wich never used mob/living so I can't use current.owner. How would I do that?